ObjCVideoQuickstart/ViewController.m

```diff
@@ -188,6 +188,49 @@ - (void)doConnect {
     [self logMessage:[NSString stringWithFormat:@"Attempting to connect to room %@", self.roomTextField.text]];
 }
 
+- (void)setupRemoteView {
+    self.remoteView = [[TVIVideoView alloc] init];
```

Comment:
Is it worthwhile to demonstrate content mode in this programmatic example, even if we just set it to AspectFit?